main.cc

```diff
@@ -1790,8 +1791,21 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
              */
             db.local().enable_autocompaction_toggle();
 
+            auto sl_migration_status = sys_ks.local().get_service_levels_migration_status().get0();
```

Comment:
would be best to put this code inside sl_controller.

ideally just in start(), guessing it's not possible due to dependencies?

Reply:
yes, it would require to pull data accessors to service_level_controlle. I also thought to do it in `qos_common.hh` but both data accessors and controller depend on it and I doing separate file only for that functions doesn't sound like enough argument.

Reply:
can be a static function if dependencies are difficult, just our main() is very long currently.

Reply:
and the code seems to be very specific SL feature logic

Reply:
created a function in `service_level_controller.hh` (outside of the class) but I had to include the data accessors in .cc file